### ISSUE TITLE
Fix BAM to FASTQ -  90513d30-5f47-473e-b180-c37a974fc03b

### ISSUE THREAD

**Issue by ke4** [ebi-ait/hca-ebi-wrangler-central#682]
SOP: https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/update_ena_runs_SOP.html
ENA project: https://www.ebi.ac.uk/ena/browser/view/PRJEB36885?show=reads

**Events:**
1. AdultHemOrgans project has 2 "datasets"/"submissions" that were archived
2. Submission 1 was archived - bam files were submitted
3. Submission 2 was archived - bam files were submitted (192 fastq = 64 bam files)
4. There were issues in the bam files. We want to replace them with the fastq files.
5. Submission 1 was re-archived - to replace the bam files with the fastq files, we ask for the suppression of bam files
"rearchived":
--- delete/"suppress" the old sequencing runs containing the bam files
--- create new sequencing runs containing the fastq files
6. All the bam files for 2 submissions were also suppressed.
7. Submission 2 files are now not available and we found out the sequencing experiment to sequencing run grouping is incorrect
--- create new sequencing experiments - 32 sequencing experiment 
--- create new sequencing runs containing the fastq files - 192 fastq files

**Problem:**
The previous sequencing runs were suppressed (in ENA). The sequencing experiments to sequencing runs linking were incorrect.

**Tasks:**
1. Submit the seq experiments. Make sure to point to correct sample accessions.
2. Submit new seq runs - use SOP

**Comment by ke4** [comment]
The list of BAM files to suppress already has been sent to ENA when we worked on this ticket: Fix BAM to FASTQ - cb156730-90b0-4b77-944c-bfc263204c61

**Comment by ke4** [comment]
Step 9 (Started) - Copy from Terra from this location: `broad-dsp-monster-hca-prod-ebi-storage/prod/455b46e6-d8ea-4611-861e-de720a562ada/data` has started into EBI cluster into `/nfs/production/hca/90513d30-5f47-473e-b180-c37a974fc03b` folder.

**Comment by prabh-t** [comment]
Alegria working on this.

**Comment by aaclan-ebi** [comment]
It looks like the files for this submission([90513d30-5f47-473e-b180-c37a974fc03b](https://contribute.data.humancellatlas.org/submissions/detail?uuid=90513d30-5f47-473e-b180-c37a974fc03b&project=455b46e6-d8ea-4611-861e-de720a562ada)) were already downloaded to the `ls /nfs/production/hca/cb156730-90b0-4b77-944c-bfc263204c61/` .

**Comment by ESapenaVentura** [comment]
Had a meeting with Alegria, we are going to re-prioritise this. 

Needs several actions:
- [ ] Determine the correct linking
- [ ] Delete the experiments associated with this submission
- [ ] Re-submit with the proper experiments/sampels/runs linking

**Comment by aaclan-ebi** [comment]
It looks like all bam files are all suppressed so I'm setting this ticket high priority now because the bam files we need to replace are no longer showing in the ENA browser.

Enrique already confirmed that the linking should be correct in Ingest. We need to re-archive this submission such that only sequencing experiments will be submitted via DSP and use the direct ENA route to submit new sequencing runs and fastq files.

**Comment by idazucchi** [comment]
@aaclan-ebi will set up a meeting with @ESapenaVentura @amnonkhen  @prabh-t  to discuss how to archive the submission

**Comment by ipediez** [comment]
@ke4 @aaclan-ebi  @ESapenaVentura and @prabh-t to have a meeting today

**Comment by jacobwindsor** [comment]
@prabh-t and @aaclan-ebi to pair

**Comment by aaclan-ebi** [comment]
Started working with @prabh-t on this. The specific steps are listed in https://docs.google.com/document/d/1xtw6kJPbcB1gDnszfZn_QFc-lCd8qnxHdbzPA-n8eQQ/edit

**Comment by ke4** [comment]
@ke4 to check if he still got the md5 values for the files

**Comment by ipediez** [comment]
@prabh-t working on this, thanks!

**Comment by ofanobilbao** [comment]
@aaclan-ebi and @prabh-t kept pairing on this yesterday. They experienced slowness. Maybe due to validation happening redundantly. @aaclan-ebi is going to create another ticket

**Comment by aaclan-ebi** [comment]
Marking this ticket as also externally blocked because of issue in DSP connection with AAP API.

**Comment by aaclan-ebi** [comment]
The AAP issue is now fixed, will work on this after the AdultLungElo archiving ebi-ait/hca-ebi-wrangler-central#724

**Comment by aaclan-ebi** [comment]
Also blocked by BioStudies validation :(

**Comment by aaclan-ebi** [comment]
There is no need to submit to BioStudies, this is not blocked by BioStudies issue in #724.

**Comment by aaclan-ebi** [comment]
The new sequencing runs and the fastq files have been submitted to ENA. They should be available soon in the ENA browser https://www.ebi.ac.uk/ena/browser/view/PRJEB36885?show=reads

The metadata entities for this project have been updated with new sequencing run accessions. The updates need to be submitted to DCP.

**Comment by gabsie** [comment]
@ESapenaVentura  - please check this and then move the ticket further

**Comment by ipediez** [comment]
The logging credentials were not working yesterday, @ESapenaVentura to try again today as yesterday there were some downtimes on ENA

**Comment by prabh-t** [comment]
No new update. ENA downtime on Friday.

**Comment by ke4** [comment]
We are going to escalate it with ENA Helpdesk.

**Comment by ofanobilbao** [comment]
@ESapenaVentura to reach out to Rasko, as he is the best contact for this

**Comment by Wkt8** [comment]
@ESapenaVentura to talk to ENA
